cpp/ql/test/library-tests/dataflow/dataflow-tests/dataflow-ir-consistency.expected

```diff
@@ -23,8 +23,12 @@ reverseRead
 argHasPostUpdate
 postWithInFlow
 | test.cpp:384:10:384:13 | memcpy output argument | PostUpdateNode should not be the target of local flow. |
+| test.cpp:384:10:384:13 | memcpy output argument | PostUpdateNode should not be the target of local flow. |
+| test.cpp:391:10:391:13 | memcpy output argument | PostUpdateNode should not be the target of local flow. |
 | test.cpp:391:10:391:13 | memcpy output argument | PostUpdateNode should not be the target of local flow. |
 | test.cpp:400:10:400:13 | memcpy output argument | PostUpdateNode should not be the target of local flow. |
+| test.cpp:400:10:400:13 | memcpy output argument | PostUpdateNode should not be the target of local flow. |
+| test.cpp:407:10:407:13 | memcpy output argument | PostUpdateNode should not be the target of local flow. |
```

Comment:
What's the reason for all these new "PostUpdateNode should not be the target of local flow." consistency check results?  Are they serious?

Reply:
It's never been a cause for concern before, no. The old IR dataflow library had many more inconsistency failures because of this, and we never saw an issue because of it.